features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/identity.xml.j2

```diff
@@ -1851,5 +1851,5 @@
     <UserFiltering>
         <ShowDisplayName>{{user_filtering.show_display_name_of_user}}</ShowDisplayName>
     </UserFiltering>
-
+<HandleLastLoginTimeAsDefaultClaim>{{account.suspension.lastLogin_as_default_claim}}</HandleLastLoginTimeAsDefaultClaim>
```

Comment:
it might be better to have this under an <AccountSuspension> tag similar to other recovery scenarios. WDYT @nilasini @IsuraD

Reply:
In previous configs, we have `<Recovery>` instead of `<AccountRecovery>`. Since this is a new config I think we can improve the configs as `<AccountSuspension>` instead of saying `<Suspension>`. WDYT?